src/client/article.ts

```diff
@@ -9,10 +9,10 @@ import setup from 'client/setup';
 import Epic from 'components/shared/epic';
 import ReactDOM from 'react-dom';
 import { ads, slideshow, videos, reportNativeElementPositionChanges } from 'client/nativeCommunication';
+import { QuizAtom } from '@guardian/atoms-rendering';
+import { QuizAtomType } from '@guardian/atoms-rendering/dist/QuizAtom';
```

Comment:
Do you think we should make this type and `QuestionType` top-level exports from `atoms-rendering`? It looks a little bit like we're using a private API here, and it worries me that someone might change it without realising it's being relied on...?